NAudio/Wave/WaveInputs/WasapiCaptureRT.cs

```diff
@@ -148,6 +148,14 @@ private void InitializeCaptureDevice(IAudioClient audioClientInterface)
             if (waveFormat == null)
             {
                 waveFormat = audioClient.MixFormat;
+                if (mix.BitsPerSample == 32)
```

Comment:
where is `mix` coming from? Not sure if I'm missing something, but doesn't look like this code will compile

Reply:
Sorry Line 150 is `var mix = audioClient.MixFormat;`